tools/ports/freetype.py

```diff
@@ -8,11 +8,20 @@
 TAG = 'version_1'
 HASH = '0d0b1280ba0501ad0a23cf1daa1f86821c722218b59432734d3087a89acd22aabd5c3e5e1269700dcd41e87073046e906060f167c032eb91a3ac8c5808a02783'
 
+variants = {'freetype-wasm-longjmp': {'SUPPORT_LONGJMP': 'wasm'}}
```

Comment:
Can we use the existing `SjLjLibrary`? https://github.com/emscripten-core/emscripten/blob/e5e97d2acf9a04731096c25094c908f0298e0724/tools/system_libs.py#L773-L804

Reply:
Not really, the way ports work and the way system libraries work is currently mostly separate.

Reply:
I see, then how about naming the library `-wasm-sjlj` to be consistent with the naming in `system_libs.py`? https://github.com/emscripten-core/emscripten/blob/e5e97d2acf9a04731096c25094c908f0298e0724/tools/system_libs.py#L793-L794